Gruntfile.js

```diff
@@ -1,84 +1,117 @@
 module.exports = function (grunt) {
   grunt.initConfig({
-    pkg: grunt.file.readJSON('package.json'),
+    pkg: grunt.file.readJSON('./package.json'),
 
     clean: {
       options: {
         force: true
       },
-      all: ['dist/']
+      all: ['./dist/', './www/']
     },
 
     fileExists: {
-      css: ['dist/css/winstrap.css']
+      css: ['./www/css/winstrap.min.css']
     },
 
     sass: {
       options: {
-        outputStyle: 'nested',
+        outputStyle: 'compressed',
         sourceMap: true,
         precision: 5,
         includePaths: [
-            "bower_components"
+            "node_modules"
         ]
       },
       dist: {
         files: {
-          'dist/css/winstrap.css': 'src/scss/winstrap.scss'
+          './dist/css/winstrap.min.css': './src/scss/winstrap.scss',
+          './dist/css/winstrap.css': './src/scss/winstrap.scss'
+        }
+      }
+    },
+    
+    //  Uglify winstrap.js
+    uglify: {
+      winstrapjs: {
+        options: {
+          beautify: true
+        },
+        files:{
+          './dist/js/winstrap.js': [

```

Comment:
No Problem. Happy to rename winstrap.js back to app.js but in commit [6c4ac73](https://github.com/edrohler/winstrap/commit/6c4ac73c50a21dbd2f7469e6e68ce7afaf237651), I concat'd jquery.js, bootstrap.js and app.js into winstrap.js/winstrap.min.js. I assumed that app.js was specific functionality to the library. I just want to make sure that app.js is only needed for the docs.